app-frontend/src/assets/styles/sass/_shame.scss

```diff
@@ -1649,6 +1645,7 @@ rf-export-item {
   box-shadow: 0px .75em 0px $shade-light, 0px 1.5em 0px $shade-light;
 }
 
+
```

Comment:
Just a weird dropdown focus outline that should probably be removed. Could be a design issue.

![image](https://user-images.githubusercontent.com/2442245/28375981-f8797470-6c76-11e7-901e-f681a8f7ecc9.png)